UnityProject/Assets/Scripts/Systems/Explosions/ExplosionNode.cs

```diff
@@ -79,5 +97,22 @@ public void Process()
 			AngleAndIntensity = Vector2.zero;
 
 		}
+
+		private SpinMode RandomSpin()
+		{
+			var num = Random.Range(0, 3);
+
+			switch (num)
+			{
+				case 0:
+					return SpinMode.None;
+				case 1:
+					return SpinMode.Clockwise;
+				case 2:
+					return SpinMode.CounterClockwise;
+				default:
+					return SpinMode.Clockwise;
+			}
+		}
```

Comment:
May I suggest moving this to some utility class? `RandomUtils` seems appropriate. I see exact same method is used in `MassDriver` and `Singularity`.

Reply:
alrighty

Reply:
resolved with commit 3170a93